pkg/controller/importconfig/manager.go

```diff
@@ -123,6 +128,27 @@ func Add(ctx context.Context,
 					UpdateFunc:  func(e event.UpdateEvent) bool { return true },
 				})),
 		).
+		Watches(
```

Comment:
Please also check if the import-controller serviceaccount in the backplane-operator has the permission.

Reply:
Good point, checked: https://github.com/stolostron/backplane-operator/blob/0d2edc3cf537a2eb4c0c9af71e1a98d104110ca0/pkg/templates/charts/toggle/server-foundation/templates/managedcluster-import-role.yaml#L73C1-L86C10